src/test/java/com/auth0/client/mgmt/JobsEntityTest.java

```diff
@@ -0,0 +1,67 @@
+package com.auth0.client.mgmt;
+
+import com.auth0.json.mgmt.jobs.Job;
+import com.auth0.net.Request;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static com.auth0.client.MockServer.*;
+import static com.auth0.client.RecordedRequestMatcher.hasHeader;
+import static com.auth0.client.RecordedRequestMatcher.hasMethodAndPath;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+
+public class JobsEntityTest extends BaseMgmtEntityTest {
+
+    @Test
+    public void shouldSendAUserAVerificationEmail() throws Exception {
+        Request<Job> request = api.jobs().sendVerificationEmail("google-oauth2|1234", null);
+        assertThat(request, is(notNullValue()));
+
+        server.jsonResponse(MGMT_JOB_POST_VERIFICATION_EMAIL, 200);
+        Job response = request.execute();
+        RecordedRequest recordedRequest = server.takeRequest();
+
+        assertThat(recordedRequest, hasMethodAndPath("POST", "/api/v2/jobs/verification-email"));
+        assertThat(recordedRequest, hasHeader("Content-Type", "application/json"));
+        assertThat(recordedRequest, hasHeader("Authorization", "Bearer apiToken"));
+
+        Map<String, Object> body = bodyFromRequest(recordedRequest);
+        assertThat(body.size(), is(1));
+        assertThat(body, hasEntry("user_id", (Object) "google-oauth2|1234"));
+
+        assertThat(response, is(notNullValue()));
+    }
+
+    @Test
+    public void shouldSendUserAVerificationEmailWithClientId() throws Exception {
+        Request<Job> request = api.jobs().sendVerificationEmail("google-oauth2|1234", "google-oauth2|client-id");
```

Comment:
I know this is just a test, but a client id value looks different. Try something similar to `AaiyAPdpYdesoKnqjj8HJqRn4T5titww` as used in another test please.